backend/server/tests/test_searchCourse.py

```diff
@@ -0,0 +1,15 @@
+import json 
+import requests 
+
+
+def test_search_course():
+    x = requests.get('http://127.0.0.1:8000/courses/searchCourse/COMP1')
+    assert x.status_code == 200
+    assert x.json().get("COMP1521") != None
+    assert x.json().get("COMP2521") == None
+    
+
+def test_invalid():
+    x = requests.get('http://127.0.0.1:8000/courses/searchCourse/ABCD')
```

Comment:
should this not be 400?

Reply:
hmm from an UX pov, when you search something that doesn't exist, I think it feels nicer to get an empty result rather than an error?